### PR TITLE
Add description to labs

### PR DESCRIPTION
**Pull request by LucasTrg**

### CODE REVIEW

File: data/labs.yaml [hunk 1]
```diff
@@ -24,6 +24,11 @@ labs:
         - Rachid
         - Guerraoui
       email: rachid.guerraoui@epfl.ch
+    description: >
+      he Distributed Computing Lab focuses currently on Scalable Implementations
```
Comment (cgrigis) [inline data/labs.yaml:28]: It looks like the prettier pass ate the first letter of several descriptions?
```suggestion
      The Distributed Computing Lab focuses currently on Scalable Implementations 
```

File: data/labs.yaml [hunk 2]
```diff
@@ -24,6 +24,11 @@ labs:
         - Rachid
         - Guerraoui
       email: rachid.guerraoui@epfl.ch
+    description: >
+      he Distributed Computing Lab focuses currently on Scalable Implementations 
+      of Cryptocurrencies, Byzantine fault tolerance and privacy in distributed machine 
+      learning, distributed algorithms making use of RDMA and NVRAM.
+      ail: rachid.guerraoui@epfl.ch
```
Comment (cgrigis) [inline data/labs.yaml:31]: Bad copy-paste? :)

```suggestion
```

File: data/labs.yaml [hunk 3]
```diff
@@ -34,6 +39,15 @@ labs:
         - Bryan
         - Ford
       email: bryan.ford@epfl.ch
+    description: >
+      he DEDIS team is working on projects related to large-scale collective
```
Comment (cgrigis) [inline data/labs.yaml:43]:
```suggestion
      The DEDIS team is working on projects related to large-scale collective
```

File: data/labs.yaml [hunk 4]
```diff
@@ -48,6 +62,29 @@ labs:
...
       email: george.candea@epfl.ch
+    description: >
+      The Dependable Systems Lab develops  techniques and abstractions for
+      building trustworthy computer systems, i.e., systems that are safe and
+      secure. They:
+
+        
+      Explore the fundamental challenges posed to security and safety by
+      large-scale systems consisting of many threads, many nodes, and millions
+      of lines of code written by many programmers
```
Comment (cgrigis) [inline data/labs.yaml:73]: Shouldn't this be part of the `<ul>` list as well?

File: data/labs.yaml [hunk 5]
```diff
@@ -48,6 +62,29 @@ labs:
...
+        </li>
+        <li>
+          Build open-source prototypes that enable evaluation of these solutions
+          with real-world workloads
+        </li>
+        <li>
+          Operate at the intersection of operating systems, distributed systems,
+          programming languages, formal methods, and computer architecture
+        </li>
+      </ul
```
Comment (cgrigis) [inline data/labs.yaml:87]:
```suggestion
      </ul>
```

File: data/labs.yaml [hunk 6]
```diff
@@ -62,6 +99,30 @@ labs:
         - Mathias
         - Payer
       email: mathias.payer@epfl.ch
+    description: >
+      ur research focuses on software and systems security. Despite efforts
```
Comment (cgrigis) [inline data/labs.yaml:103]:
```suggestion
      Our research focuses on software and systems security. Despite efforts
```

File: data/labs.yaml [hunk 7]
```diff
@@ -72,6 +133,16 @@ labs:
         - Matthias
         - Grossglauser
       email: matthias.grossglauser@epfl.ch
+    description: >
+      he Information and Network Dynamics research group is part of the
```
Comment (cgrigis) [inline data/labs.yaml:137]:
```suggestion
      The Information and Network Dynamics research group is part of the
```

File: data/labs.yaml [hunk 8]
```diff
@@ -93,6 +172,12 @@ labs:
         - Viktor
         - Kunčak
       email: viktor.kuncak@epfl.ch
+    description: >
+      Helping construct software that does what we expect
+      <br>LARA is a research group led by Viktor Kunčak. 
+      We develop precise automated reasoning techniques : tools,
+      algorithms and languages. The goal of these techniques to help
```
Comment (cgrigis) [inline data/labs.yaml:179]:
```suggestion
      algorithms and languages. The goal of these techniques is to help
```

File: data/labs.yaml [hunk 9]
```diff
@@ -107,6 +192,16 @@ labs:
         - Jean-Pierre
         - Hubaux
       email: jean-pierre.hubaux@epfl.ch
+    description: >
+      ver the last 15 years, the Laboratory for Data Security has pioneered
```
Comment (cgrigis) [inline data/labs.yaml:196]:
```suggestion
      Over the last 15 years, the Laboratory for Data Security has pioneered
```

File: data/labs.yaml [hunk 10]
```diff
@@ -119,6 +214,10 @@ labs:
         - Jean-Yves
         - Le Boudec
       email: jean-yves.leboudec@epfl.ch
+    description: >
+      he Computer Communications and Applications Laboratory 2 do research in
```
Comment (cgrigis) [inline data/labs.yaml:218]:
```suggestion
      The Computer Communications and Applications Laboratory 2 do research in
```

File: data/labs.yaml [hunk 11]
```diff
@@ -170,6 +298,13 @@ labs:
         - Touradj
         - Ebrahimi
       email: touradj.ebrahimi@epfl.ch
+    description: >
+      he Multimedia Signal Processing Group is headed by Prof. Touradj Ebrahimi.
```
Comment (cgrigis) [inline data/labs.yaml:302]:
```suggestion
      The Multimedia Signal Processing Group is headed by Prof. Touradj Ebrahimi.
```

File: data/labs.yaml [hunk 12]
```diff
@@ -180,6 +315,18 @@ labs:
         - Katerina
         - Argyraki
       email: katerina.argyraki@epfl.ch
+    description: >
+      The Network Architecture Lab works on fundamtenal questions regarding
```
Comment (cgrigis) [inline data/labs.yaml:319]:
```suggestion
      The Network Architecture Lab works on fundamental questions regarding
```

File: data/labs.yaml [hunk 13]
```diff
@@ -180,6 +315,18 @@ labs:
...
+      network system design such as : 
+      <ul>
+        <li>Can we infer network behavior from external observations?
+        <li>Can we improve Internet transparency?
+        <li>Is Internet transparency compatible with user privacy? 
+        <li>How should we write packet-processing software, such that we can reason
+        about its semantics?
+        <li>How should we write packet-processing software, such that we can reason
+        about its performance?
```
Comment (cgrigis) [inline data/labs.yaml:328]: Missing `</li>` tags

File: data/labs.yaml [hunk 14]
```diff
@@ -215,6 +393,15 @@ labs:
         - Carmela
         - Troncoso
       email: carmela.troncoso@epfl.ch
+    description:
```
Comment (cgrigis) [inline data/labs.yaml:396]: For uniformity with the other descriptions, I'd also use `>` here.

File: views/projects.tpl [hunk 15]
```diff
@@ -84,6 +84,13 @@
             <br/>
             <a href="{{ lab['url'] }}">{{ lab['name'] }}</a>
         </h1>
+        <h2>Description</h2>
+        <p>
+
```
Comment (cgrigis) [inline views/projects.tpl:89]: `{{! lab['description'] }}`?

**Review by cgrigis — CHANGES_REQUESTED** [review]
Made comments regarding some typos.
Some descriptions are in the first person (e.g. HexHive) vs 3rd person, but that can probably be reworked later. :)

**Comment by cgrigis** [comment]
It looks like this branch stems from an old commit which is not on `main`. It is good practice to rebase to `main`, in order to avoid unexpected conflicts.

I suggest the following:
- Before starting a new branch, make sure your local tree is up-to-date with `main`, and branch from there.
- Before pushing, in case `main` has changed, rebase on the latest `main` and force the push. This is acceptable because we consider branches to be "personal", i.e. you're the only person working on it. If that is not the case, coordinate with the other people working on the branch. :)

File: data/labs.yaml (hunk 14 above)
Comment (ineiti) [reply data/labs.yaml:396]: Lazy question: what is the difference in yaml if I put `:`, `:>` or `:|`?

File: data/labs.yaml [hunk 16]
```diff
@@ -215,6 +393,15 @@ labs:
...
+      The Security and Privacy Engineering Laboratory develops tools and
+      methodologies to help engineers building systems that respect societal
+      values, such as security, privacy or non discrimination. Currently, they
+      are working on ":"
+
+      M achine Learning impact on society
+      Evaluating privacy in complex systems
+      Engineering privacy-preserving systems
```
Comment (ineiti) [inline data/labs.yaml:404]:
```suggestion
      <ul>
        <li>Machine Learning impact on society</li>
        <li>Evaluating privacy in complex systems</li>
        <li>Engineering privacy-preserving systems</li>
      </ul>
```

File: views/projects.tpl [hunk 17]
```diff
@@ -93,7 +100,7 @@
             worked on in the EPFL labs affiliated with C4DT.
         </p>
         <p>
-            This page is part of the C4DT Factory that works on producing
+            This page is part of    the C4DT Factory that works on producing
```
Comment (ineiti) [inline views/projects.tpl:103]:
```suggestion
            This page is part of the C4DT Factory that works on producing
```

**Review by ineiti — CHANGES_REQUESTED** [review]
Looks good to me, thanks for the first pass. As discussed on Matrix, I'll ask the Embassy if they can have a go over the actual descriptions. I think it would be nice if we can somehow make it uniform. Something like:

- Start with a summary in one phrase of the lab, without naming the lab (as it's in the title)
- newline
- Give a longer description what the lab does, include specifically:
  - important results
  - current research areas
  - startups / spinoffs created

File: data/labs.yaml (hunk 14 above)
Comment (cgrigis) [reply data/labs.yaml:396]: https://yaml-multiline.info/  ;-)

Comment (ineiti) [reply data/labs.yaml:396]: Nice overview, thanks. The indention example doesn't seem to work, though.

**Comment by LucasTrg** [comment]
Thank you @cgrigis and @ineiti for the proof read and the suggestions ! 

We can enforce the pattern you proposed by simply expanding a bit on the template. 
I can add all of the points you added in the yaml scheme. (So that every description will have a "summary" field as well as "important results", "current research area" and "start-ups/spin offs" (we can even include URLs for the start-ups if you feel it is necessary).

**Review by cgrigis — APPROVED** [review]

**Comment by cgrigis** [comment]
Thank you for the changes, LGTM now!

**Comment by ineiti** [comment]
Sorry, I wasn't very clear. The proposition I made for the different parts of the summary are supposed to be in one block of text. No need to separate them in different tags. Just so that we know what the 'template' of the text-block is.

**Review by ineiti — APPROVED** [review]

**Pull request merged**